CHANGELOG.md

```diff
@@ -1,10 +1,13 @@
-# context_sdk Changelog
+# Changelog
 
-**[Flutter] 1.1.0 - 2024-10-16**
+All notable changes to this project are documented externally.
 
-- Update to ContextSDK 4.7.0 on iOS
+For a complete changelog, please visit: https://docs.decision.contextsdk.com/changelog/
+
+## [Flutter] 1.1.0 - 2024-10-16
 
+- Update to ContextSDK 4.7.0 on iOS
```

Comment:
I was on the fence on whether we'd want to keep maintaining this changelog manually (like how we have currently in master), or point to an external source (our `/changelog`), or both (like what I'm proposing in this PR).

WDYT @irgendeinich ? That's more up to you, I think

Reply:
Ah ok, I didn't know that 🙏 

You think we should keep a link to external docs too, just in case anyone wanna take a look at what 4.7.0 included, for instance?

Reply:
Yes - I think linking to the SDK docs as well does make sense 👍